docs/source/deploy.rst

```diff
@@ -108,25 +108,25 @@ High level instructions, specific to Voilà can be found below:
 
 1. Follow the steps of the official documentation to install the heroku
    cli and login on your machine.
-2. Add a file named runtime.txt to the project directory with the following
-   content:
+2. Add a file named runtime.txt to the project directory with a 
+   `valid Python runtime <https://devcenter.heroku.com/articles/python-support#supported-runtimes>`__:
 
    .. code:: text
 
-       python-3.7.3
+       python-3.9.9
 
 3. Add a file named Procfile to the project directory with the
    following content if you want to show all notebooks:
 
    .. code:: text
 
-       web: voila —-port=$PORT --no-browser
+       web: voila --port=$PORT --no-browser
```

Comment:
I replaced this character (and throughout the file) as it was a Unicode character (I believe an [em dash](https://www.unicodepedia.com/unicode/general-punctuation/2014/em-dash/)).